src/OTP/report_protocol.c

```diff
@@ -2010,6 +2010,20 @@ u32 Ret_u32;
 u8 cmd_lock_device (u8 * report, u8 * output)
 {
     LockDevice ();
+
+    // Erase temporary admin password
+    memset (temp_password, 0, 25);
+    tmp_password_set = 0;
+
+    // Erase temporary user password
+    memset (temp_user_password, 0, 25);
+    tmp_user_password_set = 0;
+
+    // Erase pre-authenticated message CRCs
+    authorized_crc = 0xFFFFFFFF;
+    authorized_user_crc = 0xFFFFFFFF;
+    authorized_user_crc_set=0;
```

Comment:
Could you check as well `not_authorized` variable ?

Reply:
`not_authorized` is local to the `parse_report()`-function and is handled there according to the result of the temporary key check/comparison result of the pre-authorized CRC vs. received CRC. 
I don't think we need to change anything there.